src/s1reader/s1_orbit.py

```diff
@@ -12,6 +12,10 @@
 # date format used in file names
 FMT = "%Y%m%dT%H%M%S"
 
+# Temporal margin to apply to the start time of a frame - To make sure that ANX is included when choosing the orbit file
+MARGIN_START_FRAME_SECONDS = (12*86400.0) / 175.0 + 60.0
```

Comment:
Where did these numbers came from?

Reply:
TODO: clarify what the numbers in the equations are for

Reply:
It's the orbital period of Sentinel-1 + 60 seconds. When searching for the orbit file for SAFE file, this is added to the sensing time of the frame data to make sure that the event of ascending node cross is included in the orbit file search result.